src/Options.tsx

```diff
@@ -59,6 +63,9 @@ const Options: React.FC<OptionsProps> = (props) => {
 
   const changeSizeHandle = (value: number) => {
     changeSize?.(Number(value));
+    if (onChange && typeof onChange === 'function') {
+      onChange?.(Number(value));
+    }
```

Comment:
`changeSize` and `onChange` is basicly same function, we should unify them to one.